Makefile

```diff
@@ -111,6 +111,8 @@ cover-viz: cover
 # Compile Pilosa
 build:
 	$(GO) build -tags='$(BUILD_TAGS)' -ldflags $(LDFLAGS) $(FLAGS) ./cmd/featurebase
+	$(GO) build -tags='$(BUILD_TAGS)' -ldflags $(LDFLAGS) $(FLAGS) ./cmd/fbsql
```

Comment:
@adrwalker 
> What would happen in Cloud if this fbsql wasn't built and packaged?

The packaging process would fail, this adds the fbsql executable as part of the make target so it's built every time CI builds the DB